_includes/tutorials/message-ordering/kafka/markup/test/consume-log-segments-myTopic-partition-0.adoc

```diff
@@ -0,0 +1,19 @@
+Now let's look at the Kafka broker's log segment files using the `DumpLogSegments` utility.
+First, examine partition 0, indicated by the `0` in `myTopic-0`.
+
++++++
+<pre class="snippet"><code class="shell">{% include_raw tutorials/message-ordering/kafka/code/tutorial-steps/test/dump-log-segments-0.sh %}</code></pre>
++++++
+
+You should see:
+
++++++
+<pre class="snippet"><code class="text">{% include_raw tutorials/message-ordering/kafka/code/tutorial-steps/test/expected-log-segments-myTopic-partition-0.txt %}</code></pre>
++++++
+
+Note the familiar `producerId: 0`, which corresponds to the earlier log output when we ran the producer application.
+(If the producer were not idempotent, you would see `producerId: -1`.)
+
+Also observe that each message has a unique sequence number, starting with `sequence: 0` through `sequence: 8`.
+This sequence number is how the broker detects ensures idempotency per partition (if there are retries) and ordered messages (if sequences numbers are sent out of sequence).
```

Comment:
typo "detects ensures". Also, `retries` must be > 0 if `enable.idempotence=true`